cmake/external_mlir_prebuilt.cmake

```diff
@@ -0,0 +1,62 @@
+# ******************************************************************************
+# Copyright 2017-2019 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ******************************************************************************
+
+include(ExternalProject)
+
+ExternalProject_Add(
+    ext_mlir
+    PREFIX mlir
+    URL ${MLIR_LLVM_TARBALL_URL}
```

Comment:
Where will these flags come from ? CMAKE options ? If so, please add them as CMAKE options +status message in top file.